commit ccc6432d42ee7d91f1bb9be99b93b8d0bd466551
Author: Carl Edquist <edquist@cs.wisc.edu>
Date:   Thu Jun 7 11:38:53 2018 -0500

    drop missing PACK_ENG_COMMON boilerplate
    
    and pretend we have some idea what print_linkinfo did

diff --git a/packaging_functions.sh b/packaging_functions.sh
index c4a4c3a..7dc4d8a 100644
--- a/packaging_functions.sh
+++ b/packaging_functions.sh
@@ -1,15 +1,4 @@
-if [ -z $PACK_ENG_COMMON ] ; then
-    echo "Variable PACK_ENG_COMMON undefined - this needs to point to the directory ec2_build/bin/pack-eng-common in the cdh4 branch of cdh" >&2 
-    exit 1
-fi
-if [ ! -d $PACK_ENG_COMMON ] ; then
-    echo "Directory '$PACK_ENG_COMMON' missing" >&2 
-fi
-
-for file in common.sh ; do
-[ -f $PACK_ENG_COMMON/$file ]  || exit 1
-    . $PACK_ENG_COMMON/$file
-done
+function print_linkinfo() { echo "linkinfo: $1: $2 -> $3"; }
 
 # Looks up which subdirectory of /usr/lib or ${PARCELS_ROOT}/CDH/lib a JAR is owned by
 # Outputs nothing if a symlink should not be made or the directory is unknown
